Makefile

```diff
@@ -484,15 +490,6 @@ alerts-silence:
 	${CI_DIR}/autoscaler/scripts/silence_prometheus_alert.sh BOSHJobEphemeralDiskPredictWillFill ;\
 	${CI_DIR}/autoscaler/scripts/silence_prometheus_alert.sh BOSHJobUnhealthy ;
 
-.PHONY: docker-login docker docker-image
```

Comment:
we dont crete this image anymore?

Reply:
Good catch! That is not intended to remove these lines in the context of this PR.